CHANGELOG.md

```diff
@@ -6,14 +6,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 # Upcoming
 
-### ✅ Added
-- Pipelining is enabled for requests to improve them between clients and regions over long distances. 
+### 🐞 Fixed
+- Pipelining is enabled for requests to improve them between clients and regions over long distances  [#396](https://github.com/GetStream/stream-chat-swift/pull/396)
```

Comment:
I'll try again. Let's improve the wording a little bit:
`HTTP pipelining is enabled to improve request response times for situations when the clients and servers are in different regions.`